config/filter.d/apache-common.conf

```diff
@@ -5,10 +5,16 @@
 
 # Load customizations if any available
 after = apache-common.local
+# Uncomment the following line if you use ErrorLog = syslog
+before = common.conf
```

Comment:
> Uncomment the following line...

Hmm... Is already uncommented.

Secondly, it's a bad idea to force the user to modify `.conf` (Good practice is "Never touch the stock configuration").
Better would be to write own `apache-common.local` with something like this:
```
[INCLUDES]
before = common.conf
[DEFAULT]
_apache_prefix = %(__prefix_line)s
```